### PR TITLE
WIP: 10 minute tutorial java gradle

### PR DESCRIPTION
**Pull request by Riggs333**
Enhance 10 minute tutorial with Gradle examples and console outputs.
:construction:

### CODE REVIEW

File: content/docs/guides/10-minute-tutorial.md [hunk 1]
```diff
@@ -177,6 +177,8 @@ public class RunCucumberTest
 }
 ```
 
+Create an empty directory `src/test/resources/hellocucumber` which later will contain the feature files.
```
Comment (mlvandijk) [inline content/docs/guides/10-minute-tutorial.md:180]: Afaik this directory should be created by the archetype?

**Review by mlvandijk — COMMENTED** [review]
I realise this is WIP, but one minor comment.

File: content/docs/guides/10-minute-tutorial.md (hunk 1 above)
Comment (anna-engle) [reply content/docs/guides/10-minute-tutorial.md:180]: Just wanted to mention, there was a bug open for this pieceof the tutorial (https://github.com/cucumber/docs.cucumber.io/issues/367) and I opened a PR: https://github.com/cucumber/docs.cucumber.io/pull/425

**Comment by mlvandijk** [comment]
@Riggs333 Could you please update your branch in line with master, to take @annabaker's changes into account. Or should this PR be abandoned?

**Comment by Riggs333** [comment]
Sorry for my late answer.
Unfortunately I don't have much time to follow up on this in the near future.
And it looks like these docs are evolving pretty good along other PRs.
The changes of this PR here are not that big anyway. ;-)

So I'll close this PR.

**Pull request closed without merging**